components/core/src/clp_s/search/OutputHandler.cpp

```diff
@@ -1,6 +1,54 @@
 #include "OutputHandler.hpp"
 
+#include <sstream>
+
+#include <spdlog/spdlog.h>
+
 namespace clp_s::search {
+NetworkOutputHandler::NetworkOutputHandler(
+        std::string const& host,
+        std::string const& port,
+        bool should_output_timestamp
+)
+        : OutputHandler(should_output_timestamp) {
+    struct addrinfo hints = {};
```

Comment:
I think this more or less replicates the now-deleted clo search controller connection code right? I also duplicate this code in my follow-up reducer PR. Do you think it makes sense to move this into into clp/networking/socket_utils.hpp to pre-emptively deduplicate this generic tcp connection code, or should we leave that for later?